dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -824,6 +825,24 @@ static boolean doesOrganizationExistToUser(Long organizationId, Long userId, Org
     }
 
 
+    static boolean isUserAdminOrMaintainer(Organization organization, Long userId) {
+        OrganizationUser organizationUser = getUserOrgRole(organization, userId);
```

Comment:
I think you have to check for a null organization here. This is where I'm a fan of `Optional`, BTW, because it makes it clear when you do have to check and when you don't (but you don't have to do that)